install/kubernetes/helm/meshery-operator/crds/crds.yaml

```diff
@@ -131,30 +134,38 @@ spec:
                   custom:
                     properties:
                       url:
+                        description: URL of the custom Broker to connect to
                         type: string
                     type: object
                   native:
                     properties:
                       name:
+                        description: Name of the native Broker to connect to
                         type: string
                       namespace:
+                        description: Namespace where the native Broker is deployed
```

Comment:
Is there a default namespace that we would commonly expect to see? Can this be any namespace?

Reply:
The namespace here is that of the Broker instance to be deployed.